dev-docs/bidders/zeta_global_ssp.md

```diff
@@ -7,6 +7,10 @@ biddercode: zeta_global_ssp
 bidder_supports_deals: false
 media_types: banner
 gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+userIds: sharedId
```

Comment:
I thought you pulled in all eids

Reply:
Sorry, right, my mistake. Changed to **all**